drivers/python/requirements-test.txt

```diff
@@ -0,0 +1,4 @@
+PyOpenSSL
+PyYAML
+twisted
+tornado
```

Comment:
Why not just include these as [`deps`](https://testrun.org/tox/latest/config.html#confval-deps=MULTI-LINE-LIST)? That would even allow for not including things like `twisted` where they are not compatible.